migrations/datastore/mysql/000001_init.up.sql

```diff
@@ -4,6 +4,7 @@ CREATE TABLE IF NOT EXISTS objectType (
   id int NOT NULL AUTO_INCREMENT,
   typeId varchar(64) NOT NULL,
   definition json DEFAULT NULL,
+  ALTER TABLE table_name MODIFY COLUMN definition JSON NOT NULL;
```

Comment:
This isn't a legal sql query and will break the migration as you can't alter a table within a create table statement.

In addition, migrations shouldn't be modified as that will break existing deployments. You'll instead need to create a brand new migration that alters the column to make it not null (and perhaps includes a default value).